tests/basic.rs

```diff
@@ -273,3 +273,25 @@ async fn address_send_exercises_backpressure() {
         .now_or_never()
         .expect("be able to queue another message because the mailbox is empty again");
 }
+
+#[tokio::test]
+async fn address_debug() {
```

Comment:
Technically, this doesn't need to be an `async` test.

Reply:
Good point